tools/clang/test/SemaHLSL/hlsl/intrinsics/barrier/barrier-node-errors.hlsl

```diff
@@ -192,3 +192,12 @@ void node02(RWThreadNodeInputRecord<RECORD> input,
   Barrier(threadRec, DEVICE_SCOPE);
   threadRec.OutputComplete();
 }
+
+// Make sure default broadingcast launch is supported when dignose barrier.
```

Comment:
As there's no expected errors here, maybe some more explanation would help about what this is testing and why? I think there's maybe something about the `Barrier` that means it'll only work if the NodeLaunch type is broadcasting?

Reply:
Updated the comment.